.travis.yml

```diff
@@ -1,39 +1,28 @@
 language: node_js
-
 node_js:
-  - "0.10"
-  - "0.12"
-
+- '0.10'
```

Comment:
Fixed. I was using travis commands to add the secure keys, and it mucked with the formatting.